src/main/java/org/folio/marccat/resources/SearchAPI.java

```diff
@@ -58,7 +58,38 @@ public SearchResponse search(
     }, tenant, okapiUrl, configurator);
   }
 
+  @GetMapping("/searchAuth")
+  public SearchResponse searchAuth(
+    @RequestParam(name = "lang", defaultValue = "eng") final String lang,
+    @RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) final String tenant,
+    @RequestHeader(Global.OKAPI_URL) String okapiUrl,
+    @RequestParam("q") final String q,
+    @RequestParam(name = "from", defaultValue = "1") final int from,
+    @RequestParam(name = "to", defaultValue = "10") final int to,
+    @RequestParam(name = "ml", defaultValue = "170") final int mainLibraryId,
```

Comment:
`170` for this default seems a bit magical, how is it determined?

Reply:
There is dummy data for holdings and using library 170 by default. When we develop the holdings part the data will be collected from settings.

Reply:
Thanks for the info. I think a comment in the PR or the code (preferably in the code) to indicate which data is dummy data would be useful for reviewers.